specs/latest/2.0/index.html

```diff
@@ -1498,28 +1624,54 @@ <h2 class="no-toc">Table of contents</h2>
         </p>
       </dd>
 
-      <dt class="idl-code">void texImage3D(GLenum target, GLint level, GLint internalformat, GLsizei width, GLsizei height, GLsizei depth, GLint border, GLenum format, GLenum type, ArrayBufferView? pixels)
+      <dt class="idl-code">
+        void texImage3D(GLenum target, GLint level, GLint internalformat, GLsizei width,
+                        GLsizei height, GLsizei depth, GLint border, GLenum format, GLenum type,
+                        ArrayBufferView srcData, GLuint srcOffset)
```

Comment:
We discussed this offline and two entry points are needed here, since passing "null" is useful to allocate a zeroed-out texture.